pkgs/applications/networking/listadmin/default.nix

```
@@ -0,0 +1,50 @@
+{ lib, stdenv, fetchurl, makeWrapper, perlPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "listadmin";
+  version = "2.73";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/listadmin/${version}/listadmin-${version}.tar.gz";
+    sha256 = "00333d65ygdbm1hqr4yp2j8vh1cgh3hyfm7iy9y1alf0p0f6aqac";
+  };
+
+  buildInputs = [ perlPackages.perl ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  configurePhase = ''
+  '';
+
+  buildPhase = ''
+    true
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    install -m 755 listadmin.pl $out/bin/listadmin
+    gzip -c listadmin.1 > $out/share/man/man1/listadmin.1.gz
+
+    wrapProgram $out/bin/listadmin \
+      --prefix PERL5LIB : "${with perlPackages; makePerlPath [
```

This ensures perl and perlPackages are the same when overriding perl:

```suggestion
      --prefix PERL5LIB : "${with perl.pkgs; makePerlPath [
```